src/components/NWCBudgetEditor.tsx

```diff
@@ -31,29 +31,49 @@ export type BudgetForm = {
 export function NWCBudgetEditor(props: {
     initialProfile?: NwcProfile;
     initialName?: string;
+    initialAmount?: string;
+    initialInterval?: "Day" | "Week" | "Month" | "Year";
     onSave: (value: BudgetForm) => Promise<void>;
 }) {
     const i18n = useI18n();
 
+    console.log(props);
+
+    const connection_name = props.initialProfile?.name
+        ? props.initialProfile?.name
+        : props.initialName
+        ? props.initialName
+        : "";
```

Comment:
same here, think this can be simplified as 

`props.initialProfile?.name || props.initialName || "`